onnxruntime/core/providers/cpu/tensor/pad.cc

```diff
@@ -247,6 +275,51 @@ Status PadCpuImpl<float>(OpKernelContext* ctx,
 
 template <>
 Status Pad<float>::Compute(OpKernelContext* ctx) const {
-  return PadCpuImpl<float>(ctx, pads_, slices_, mode_, value_);
+  // kOnnxDomain Pad opset >= 11 (Or) kMsDomain opset == 1
+  if (is_dynamic_) {
+    const Tensor& input_tensor = *ctx->Input<Tensor>(0);
+    std::vector<int64_t> output_dims(input_tensor.Shape().GetDims());
+    size_t dimension_count = output_dims.size();
+
```

Comment:
Why do we need to copy the dims if you only need the size?

Reply:
didn't notice this. Thanks, will fix.

Reply:
Fixed.